### ISSUE TITLE
Running demo on PhoneGap 

### ISSUE THREAD

**Issue by samasama-studios** [godesign/nativeDroid2#39]
Hello guys,

First of all thanks and congratulations for the amazing template.

Do you have any guideline to build the demo on PhoneGap wrapper? Theme does not seems to be loaded – http://i.imgur.com/3rwOSLw.png

I am using a clean working copy of the project, a freshly created PhoneGap project and placing the entire demo content within the www folder.

Any input highly appreciated!

José

**Comment by patrickjauernig** [comment]
Check if the paths inside the CSS/HTML files are correct. Github strips out the "http" prefixes, so the browser is unable to load JQuery etc.

**Comment by mustafacoskuncom** [comment]
Hi Guys, 
I have same issue I tried a few ways like download the .js & .css file in my local project however not work as on computer broswer? Its normally working on the comp. but in Cordova and build on Mobile Device not working... Anyone test it on the real Android Device or VM? Thanks again,

**Comment by godesign** [comment]
As @patrickjauernig already suggested did you adjusted and check the paths to the .js and .css files?

**Comment by mustafacoskuncom** [comment]
Yes tried a few ways but not works on phonegap. And another paradox works on computer browser. Under normal condition as you know everything is nearly same just put in www folder and run, am I right? Thanks again..

**Comment by godesign** [comment]
I'm not a phonegap/cordova expert, but as far as I know you have to link everything locally in the www-folder (Also the vendors loaded from CDN's - jquery/jqm/jqUI). You might also have a look at the stylesheets loaded by import here: https://github.com/godesign/nativeDroid2/blob/master/css/nativedroid2.css

As a further resources I suggest have a look at this SO question:
http://stackoverflow.com/questions/21479768/resource-structure-in-apache-cordova-phonegap-and-loading-the-files-from-the-f

**Comment by mustafacoskuncom** [comment]
ı did all files localized cause I think maybe couldnt get from internet however not work but I ll work on it if solve share here. Thanks again...

**Comment by kulu15** [comment]
hello guys!
you have to start nativedroid with cordova? I have the same problem you

**Comment by mustafacoskuncom** [comment]
Yes I did. You should localized all sources and then open the files in your code editor and redesign the path but be sure use "/path/path" or use "path/path" sometimes it can efffect the paths... Except of these you need nt extreme changes... Thanks.

**Comment by kulu15** [comment]
Thank you for the answer @mustafacoskuncom ; I had already changed the way as you said. In the PC browser the result is good but it is at the time of testing on my android phone as it does not work ... the visual is ugly

**Comment by mustafacoskuncom** [comment]
I did it and it seems great nearly native. I ll develope it for my own app. Be Carefull maybe forget something ;)

**Comment by kulu15** [comment]
Thanks !
also had to change the way in cdn http:

**Comment by rodrigoclp** [comment]
Change paths: //... to http:// AND /local_path/... to local_path/...
Please, change definitely on repository!
Thanks a lot for this good work.

**Comment by Phinrich** [comment]
Hi and a BIG thank you for an AWSOME Template. I am using the Autocomplete which, upon succesfull selection, opens the search-list.html file. I would like to open my own html file  at this point. Apart from re-naming my file "search-list.html" I cant seem to find where in your code the destination file is specified. Any help would be appreciated.  Thanks

Paul

**Comment by Phinrich** [comment]
Having said that I must admit to being surprised to see the search-list.html page as I had run the code at least 50 times before the search-list.html file popped up seemingly out of the blue. One of the idiosyncrasies of JavaScript I suppose.

Paul

**Comment by Phinrich** [comment]
Can I get two searches out of the Autocomplete Search an can I distinguish between the results ?

Thanks

Paul
